src/Execution/ExecutionStrategy.php

```diff
@@ -138,19 +212,20 @@ private function getFieldNameKey(FieldNode $node)
     }
 
     /**
-     * Implements the "Evaluating selection sets" section of the spec
-     * for "read" mode.
-     * @param ObjectType $parentType
-     * @param            $source
+     * Implements the "Evaluating selection sets" section of the spec for "read" mode.
+     * @param ObjectType $objectType
+     * @param            $rootValue
      * @param            $path
      * @param            $fields
-     *
      * @return array
      * @throws InvalidTypeException
+     * @throws \Digia\GraphQL\Error\ExecutionException
+     * @throws \Digia\GraphQL\Error\InvariantException
+     * @throws \Throwable
```

Comment:
Where does \Throwable come from?

Reply:
![screen shot 2018-03-15 at 1 06 15](https://user-images.githubusercontent.com/199614/37435715-18ac5672-27ed-11e8-8e16-a227cb32aa9e.png)
It's deep down somewhere, PhpStorm suggested to add that tag.